src/common/time/TimeUtils.cpp

```diff
@@ -25,7 +25,8 @@ constexpr int64_t kMaxTimestamp = std::numeric_limits<int64_t>::max() / 10000000
       return Status::Error("Invalid value type.");
     }
     if (kv.first == "year") {
-      if (kv.second.getInt() < std::numeric_limits<int16_t>::min() ||
+      // year should be in [-32767, 32767] and same as parser
+      if (kv.second.getInt() <= std::numeric_limits<int16_t>::min() ||
```

Comment:
Why less and equal?

Reply:
The max of year in paser is 32767. The min is -32767.
In order to be same as parser.

Reply:
Where limit this in parser?

Reply:
The year, month, day, hour and others are INTEGER.
INTEGER is a `[0-9]+` and cannot be a negative int.
Negative year supports by 
```
| NEGATIVE INTEGER NEGATIVE INTEGER NEGATIVE INTEGER {
    $$ = new nebula::Date(0-$2, $4, $6);
```

Reply:
I can't catch you.

Reply:
fixed. year is in [-32768,32767].